Automattic/wp-calypso
# Domains: Fix the set primary domain notice on success/error

#### Changes proposed in this Pull Request

* We're removing the static notice (and the undo button) in favour of standard success notice for consistency with the rest of Calypso

#### Testing instructions

* on the domains list screen click on the "Change primary" button and verify it's working and a standard success notice is shown
